src/components/ResourceTypePicker/index.js

```diff
@@ -39,30 +40,37 @@ const ResourceTypePicker = ({
   resourceTypeFilters,
   selectResourceTypeAction,
   selectedResourceType,
-}) => (
-  <View>
-    <ScrollView
-      style={styles.root}
-      horizontal
-      showsHorizontalScrollIndicator={false}
-      contentContainerStyle={styles.contentContainerStyle}
-    >
-      {
-        resourceTypeFilters
-          .filter(({ typeIsEnabled }) => typeIsEnabled === true)
-          .map(({ type, label }) => (
-            <CategoryButton
-              key={type}
-              resourceType={type}
-              label={label}
-              isActive={selectedResourceType === type}
-              selectResourceTypeAction={selectResourceTypeAction}
-            />
-          ))
-      }
-    </ScrollView>
-  </View>
-);
+}) => {
+  // select first resourceType on initial load
+  useEffect(() => {
+    selectResourceTypeAction(resourceTypeFilters[0].type);
```

Comment:
* when switching between multiple collections, this appears to select "Encounters," regardless of what the previously-selected type was for that collection.  ...should it actually select the previously-active type?

* when "Encounters" is turned off via the filter drawer, no type is selected.  (should the next one be selected, or: last-active?)

Reply:
Refactored to select the first `resourceTypeFilter` with `typeIsEnabled === true`